### PR TITLE
11.0 Merge from 10.0

### PR DESCRIPTION
**Pull request by fedoranvar**

### CODE REVIEW

**Review by Ramil-Mukhametzyanov — CHANGES_REQUESTED** [review]
OSError: File not found: openapi/views/tour_views.xml

**Review by Ramil-Mukhametzyanov — CHANGES_REQUESTED** [review]
image

File: openapi/doc/changelog.rst [hunk 1]
```diff
@@ -1,3 +1,13 @@
+`1.1.2`
+-------
+
+- **Improvement:** Security rules for reading and configuration
```
Comment (Ramil-Mukhametzyanov) [inline openapi/doc/changelog.rst:4]: :+1: checked

File: openapi/doc/changelog.rst [hunk 2]
```diff
@@ -1,3 +1,13 @@
...
+
+`1.1.1`
+-------
+
+- **Fix:** Translation model's float-field into JSON's integer
```
Comment (Ramil-Mukhametzyanov) [inline openapi/doc/changelog.rst:9]: :+1: checked

**Review by Ramil-Mukhametzyanov — APPROVED** [review]

**Review by yelizariev — APPROVED** [review]

**Comment by yelizariev** [comment]
I approve to merge it now

**Pull request merged**